.github/renovate.json

```diff
@@ -33,7 +33,7 @@
     },
     {
```

Comment:
automerge can be enabled here

Reply:
Realized the same, at about the same time :P https://github.com/hashintel/hash/pull/2892/commits/ab68632bab9a12b03ade53d23d4c50573cde7b38